apps/obs-web/src/views/ViewImage.vue

```diff
@@ -16,65 +16,78 @@ import {
   watch,
   onBeforeUnmount,
   onMounted,
-  onServerPrefetch
+  onServerPrefetch,
 } from '@vue/composition-api';
-import { Vue } from 'vue-property-decorator';
-import { CouchDBInfo, CouchDBCredentials, couchService } from '@boatnet/bn-couch';
+import {
+  CouchDBInfo,
+  CouchDBCredentials,
+  couchService,
+} from '@boatnet/bn-couch';
 import { Client, CouchDoc, ListOptions } from 'davenport';
+import Vue, { WatchOptions } from 'vue';
 
 export default createComponent({
-    setup(props, context) {
-
-        const id: any = ref(context.root.$route.params.id ? context.root.$route.params.id : 0);
-        const transferring: any = ref(false);
-        onMounted( async () => {
-            if (id.value) {
-                const masterDB: Client<any> = couchService.masterDB;
-                const queryOptions = {
-                    reduce: false,
-                    include_docs: true,
-                    attachments: true,
-                    key: id.value
-                };
-                transferring.value = true;
-                const response: any = await masterDB.view<any>(
-                    'obs_web',
-                    'ots_trips_with_logbook_captures',
-                    queryOptions
-                );
+  props: {
```

Comment:
yes, I was trying to make use of this component for EM Review. When navigating to the EM Review page we use tripNum, so that wouldn't work for this component. So moved id to props to make it more generic and usable if other components need to use it too. Checked and the Image button on task management still works.

Reply:
makes sense.  thanks.